### PR TITLE
Changing license headers

### PR DESCRIPTION
**Pull request by tomikazi**

### CODE REVIEW

**Comment by tomikazi** [comment]
retest this please

**Review by adibrastegarnia — APPROVED** [review]

File: Makefile [hunk 1]
```diff
@@ -11,40 +15,33 @@ build:
...
+	go run cmd/onos-cli-docs-gen/main.go
+
 jenkins-test:  # @HELP run the unit tests and source code validation producing a junit style report for Jenkins
-jenkins-test: build-tools deps license_check linters
+jenkins-test: mod-lint build license_check_apache linters license
 	TEST_PACKAGES=github.com/onosproject/onos-cli/... ./../build-tools/build/jenkins/make-unit
 
 coverage: # @HELP generate unit test coverage data
 coverage: build deps linters license_check
 	./../build-tools/build/coveralls/coveralls-coverage onos-cli
```
Comment (adibrastegarnia) [inline Makefile:44]: do you want to add a target for doc generator? doesn't need to be called but for ease of use

Comment (tomikazi) [reply Makefile:44]: I've added is separately, not as part of this.

**Comment by ray-milkey** [comment]
retest this please

**Review by ray-milkey — APPROVED** [review]

File: Makefile [hunk 2]
```diff
@@ -11,39 +15,28 @@ build:
...
-	golangci-lint --version || curl -sfL https://install.goreleaser.com/github.com/golangci/golangci-lint.sh | sh -s -- -b `go env GOPATH`/bin v1.42.0
+docs: # @HELP generate CLI docs
+docs:
+	go run cmd/onos-cli-docs-gen/main.go
 
-license_check: # @HELP examine and ensure license headers exist
-	@if [ ! -d "../build-tools" ]; then cd .. && git clone https://github.com/onosproject/build-tools.git; fi
-	./../build-tools/licensing/boilerplate.py -v --rootdir=${CURDIR}
+jenkins-test:  # @HELP run the unit tests and source code validation producing a junit style report for Jenkins
+jenkins-test: mod-lint build license_check_apache linters license
```
Comment (ray-milkey) [inline Makefile:38]: Can safely remove the license_check_apache target here

**Review by ray-milkey — CHANGES_REQUESTED** [review]

**Review by ray-milkey — APPROVED** [review]

**Pull request merged**